nano/node/lmdb.cpp

```diff
@@ -717,8 +760,9 @@ blocks_info (0),
 representation (0),
 unchecked (0),
 vote (0),
-meta (0),
-stopped (false)
+online_weight (0),
+stopped (false),
+meta (0)
```

Comment:
meta and stopped have swapped order, so they now no longer match the same order in the class definition.

Reply:
Probably happened during my rebase.

Reply:
I rebased after your PR to master, @wezrule . online_weight { 0 } is now in header file

Reply:
ok thanks